python/tk_3dsmax/menu_generation.py

```diff
@@ -141,19 +141,7 @@ def _jump_to_fs(self):
         # launch one window for each location on disk
         paths = self._engine.context.filesystem_locations
         for disk_location in paths:
-            # get the setting
-            system = sys.platform
-
-            # run the app
-            if system == "linux2":
```

Comment:
Copy pasted from the nuke engine, we only need the win32 implementation for max.